### PR TITLE
[#149] Video Player Component

### PR DESCRIPTION
**Pull request by bd-viget**
# Summary

This PR adds the custom Video Player component

## Issues

* #149 

## Testing Instructions

1. Create a new page
2. Add the Video Player component
3. Save changes
4. Test on Frontend

### CODE REVIEW

File: wp-content/themes/wp-starter/blocks/video-player/style.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+		p, > .wp-block-group {
+			@apply mt-16 mx-auto max-w-[var(--wp--style--global--wide-size)];
+		}
+
+		.wp-block-cover {
+			@apply relative !z-20 h-full mt-0;
+		}
+
+		> .wp-block-group {
+			@apply relative overflow-hidden aspect-video max-w-[1000px] mx-auto;
```
Comment (nathan-schmidt-viget) [inline wp-content/themes/wp-starter/blocks/video-player/style.css:43]: Do we need to set a max width?

**Review by nathan-schmidt-viget — APPROVED** [review]
Looks great!

File: wp-content/themes/wp-starter/blocks/video-player/template.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      }
+    ],
+    [
+      "core/group",
+      [],
+      [
+        [
+          "core/cover",
+          {
+            "className": "video-overlay-content",
```
[the remark this answers is not in the document]
Comment (bd-viget) [reply wp-content/themes/wp-starter/blocks/video-player/template.json:24]: I don't think we do!

File: wp-content/themes/wp-starter/blocks/video-player/style.css (hunk 1 above)
Comment (bd-viget) [reply wp-content/themes/wp-starter/blocks/video-player/style.css:43]: Hmmm. I guess not!

**Pull request merged**